docs/source/commands/graphs.mdx

```diff
@@ -63,7 +63,7 @@ You can also save the output to a local `.graphql` file like so:
 
 ```bash
 # Creates prod-schema.graphql or overwrites if it already exists
-rover graph fetch my-graph@my-variant > prod-schema.graphql
```

Comment:
i also went through the docs and replaced all usage of `>` with `--output` 😃

Reply:
yaaaaas